proto/bragi/common.proto

```diff
@@ -49,25 +49,40 @@ message ControlErrorAnnouncement {
   optional string message = 2;
 }
 
+message MatchTimeline {
+  repeated Match matches = 1;
+}
+
 message Match {
   string match_urn = 1;
-  string tournament_urn = 2;
+  string tournament_urn = 2 [deprecated=true]; // Deprecated. Use tournament.
   Sport sport = 3;
-  string home_team_urn = 4;
-  string away_team_urn = 5;
+  string home_team_urn = 4 [deprecated=true]; // Deprecated. Use teams.
+  string away_team_urn = 5 [deprecated=true]; // Deprecated. Use teams.
   uint32 home_score = 6;
   uint32 away_score = 7;
   MatchStatusType match_status = 8;
   google.protobuf.Timestamp plannet_start = 9;
+  Tournament tournament  = 10;
+  repeated Team teams = 11;
+  repeated Player players = 12;
 }
 
-message MatchTimeline {
-  repeated Match matches = 1;
+message Tournament {
+  string urn = 1;
+  string name = 2;
 }
 
 message Team {
-  string team_urn = 1;
-  string team_name = 2;
+  string urn = 1; // id is urn in format e.g. 'od:team:123'
+  string name = 2;
+  bool is_home = 3;
+}
+
+message Player {
+  string urn = 1; // id is urn in format e.g. 'od:player:123'
```

Comment:
obsolete `id is urn in format e.g. 'od:player:123'`

Reply:
Fixed